### ISSUE TITLE
Reduce ServerToken Info Output and Disable ServerSignature

### ISSUE THREAD

**Issue by binaryanomaly** [nextcloud/nextcloud-snap#612]
The apache server should by default not expose so much detailed information that gives a potential attacker valuable information about the exact versions in use:

> Date: Thu, 14 Jun 2018 15:37:19 GMT
> Server: Apache/2.4.33 (Unix) OpenSSL/1.0.2g
> X-Powered-By: PHP/7.1.17

```
ServerTokens Prod
ServerSignature Off 
```
Should be part of the default configuration especially since they cannot be changed by the user in a snap setup.

**Comment by kyrofa** [comment]
Yes, we need to hide this information behind a debug configuration flag. There are other similar things as well.
